modules/templates/helper.go

```diff
@@ -286,6 +286,9 @@ func NewFuncMap() []template.FuncMap {
 			}
 			return false
 		},
+		"svg": func(icon string, size int) template.HTML {
```

Comment:
If the icons.svg sprite will only contain octicon. I would suggest using `svg-octicon` as the helper name. Futhermore auto-prefix with octicon- the hash to not have repated at each calls.

Reply:
This was changed per a discussion with @silverwind in order to make it more generic, since future SVGs can be added to the same sprite to reduce HTTP calls.

Reply:
We want to support future icon libraries so it is named generically so we can have `fa-lock` later.